ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_stat.py

```diff
@@ -131,3 +131,34 @@ def test_paddle_median(
         axis=axis,
         keepdim=keepdim,
     )
+
+
+# var
+@handle_frontend_test(
+    fn_tree="paddle.var",
+    dtype_and_x=_statistical_dtype_values(function="var"),
+    unbiased=st.booleans(),
+    keepdim=st.booleans(),
+)
+def test_paddle_var(
+    *,
+    unbiased,
+    dtype_and_x,
+    keepdim,
+    fn_tree,
+    frontend,
+    backend_fw,
+    test_flags,
+):
+    input_dtype, x, axis = dtype_and_x
```

Comment:
The issue is here when unpacking values this helper function returns tuple of 4 variables but you're unpacking only 3
```suggestion
    input_dtype, x, axis, _ = dtype_and_x
```